fluffy/templates/paste.html

```diff
@@ -31,13 +31,29 @@
 {% endblock %}
 
 {% block text %}
-    <div class="line-numbers">
-        {% for i in range(1, num_lines(text) + 1) %}
-            <a id="LL{{i}}">{{i}}</a>
-        {% endfor %}
-    </div>
-    <div class="text" contenteditable="true" spellcheck="false">
-        {{highlighter.highlight(text)|safe}}
+    <div style="display:flex;">
+        <div style="flex:1; width:30%;">
+            <div class="line-numbers" style=>
```

Comment:
Loose `style=` here?

```suggestion
            <div class="line-numbers">
```